integration_tests/helpers/s3Helper.ts

```diff
@@ -157,15 +157,12 @@ interface S3BillingStandardised {
 const s3GetObjectsToArray = async (
   bucketName: string,
   folderPrefix: string
-): Promise<S3BillingStandardised[]> => {
+): Promise<BillingStandardised[]> => {
   const s3Response = await getAllObjectsFromS3(bucketName, folderPrefix);
-  const convertS3Repsonse2Str = JSON.stringify(s3Response);
-  const formatS3Str = convertS3Repsonse2Str
-    .replace(/:[^"0-9.]*([0-9.]+)/g, ':\\"$1\\"') // converts digits to string for parsing
-    .replace(/\\n|'/g, "") // removes //n character , single quotes
-    .replace(/}{/g, "},{"); // replace comma in between }{ brackets
-  const data = JSON.parse(formatS3Str);
-  return JSON.parse("[" + String(data["0"]) + "]");
+  console.log("🚀 ~ file: s3Helper.ts:162 ~ s3Response", s3Response)
+  const s3String = s3Response.join("").replace(/\n/g,"").replace(/}{/g,'},{')
+  console.log("🚀 ~ file: s3Helper.ts:163 ~ s3String", s3String)
```

Comment:
This line-number thing seems hard to maintain (and it's already off), do we need this "logging" here?

Reply:
no was using for debugging missed to remove it earlier, removed now